docs/data/joy/components/drawer/drawer.md

```diff
@@ -1,26 +1,23 @@
 ---
 productId: joy-ui
 title: React Drawer component
+components: Drawer
 githubLabel: 'component: drawer'
 ---
 
 # Drawer
 
 <p class="description">Navigation drawers provide quick access to other destinations in your app without removing the user out of context.</p>
 
-:::info
-💡 The Joy UI Drawer component is still in development.
-If you're in need of it, please upvote [**this GitHub issue**](https://github.com/mui/material-ui/issues/36292) to help us prioritize the next batch of new components.
-:::
+## Introduction
 
-## Using the Modal component
+{{"demo": "DrawerUsage.js", "hideToolbar": true, "bg": "gradient"}}
 
-In the meantime, you can build your own Drawer using Joy UI's [Modal](/joy-ui/react-modal/) and [Sheet](/joy-ui/react-sheet/) components as a starting point.
-The demo below shows how to do it.
+## Basic
 
-<iframe src="https://codesandbox.io/embed/drawer-joy-ui-2s5f3w?module=%2FDrawer.tsx&fontsize=14&hidenavigation=1&theme=dark&view=preview"
-     style="width:100%; height:500px; border:2; border-radius: 8px; overflow:hidden;"
-     title="Drawer - Joy UI"
-     allow="accelerometer; ambient-light-sensor; camera; encrypted-media; geolocation; gyroscope; hid; microphone; midi; payment; usb; vr; xr-spatial-tracking"
-     sandbox="allow-forms allow-modals allow-popups allow-presentation allow-same-origin allow-scripts"
-   ></iframe>
+The navigation drawers can toggle open or closed. Closed by default, the drawer opens temporarily above all other content until a section is selected.
+
+The Drawer can be cancelled by clicking the overlay or pressing the Esc key.
+It closes when an item is selected, handled by controlling the `open` prop.
+
+{{"demo": "DrawerBasic.js"}}
```

Comment:
```suggestion

## Customization

### Anchor position

{{"demo": "DrawerAnchor.js"}}
```

I think the DrawerBasic can be simpler with the default anchor. Maybe an 
`<IconButton><MenuIcon /></IconButton>` that toggle the Drawer.

and then add another demo in the customization for changing the anchor.

Reply:
Restructured a bit the demos:
- the basic containes simple app bar & sandwich that open drawer - this is the most common use-case
- the customization has demos for the different anchor & sizes that can be used on the drawer.